desktop/dev-environments.md

```diff
@@ -150,8 +150,8 @@ You can also use Dev Environments to collaborate on any Docker Compose-based pro
   >
   > When cloning a Git repository using SSH, ensure you've added your SSH key to the ssh-agent. To do this, open a terminal and run `ssh-add <path to your private ssh key>`.
 
-1. Click **Create** to open the **Create a Dev Environment** dialog. Then, copy `https://github.com/dockersamples/compose-dev-env.git` and add it to the **Docker image** field on the **Remote** tab.
-2. Click **Create**. This initializes the project and clones the Git code and builds the Compose application. This:
+1. Under Dev Environments, click **Create**. This opens the **Create a Dev Environment** dialog. Click **Get Started** and then copy `https://github.com/dockersamples/compose-dev-env.git` and add it to the **Enter the Git Repository** field on the **Existing Git repo** source.
+2. Click **Continue**. This initializes the project and clones the Git code and builds the Compose application. This:
```

Comment:
Not a show-stopper, but perhaps we should split the second `Click` "action item" to its own line

```suggestion
1. Under Dev Environments, click **Create**. This opens the **Create a Dev Environment** dialog.
2. Click **Get Started** and then copy `https://github.com/dockersamples/compose-dev-env.git` and add it to the **Enter the Git Repository** field on the **Existing Git repo** source.
3. Click **Continue**. This initializes the project and clones the Git code and builds the Compose application. This:
```

(perhaps `2.` could be split even further, but haven't tried how that looks);

```suggestion
1. Under Dev Environments, click **Create**. This opens the **Create a Dev Environment** dialog.
2. Click **Get Started**
3. Copy `https://github.com/dockersamples/compose-dev-env.git` and add it to the **Enter the Git Repository** field on the **Existing Git repo** source.
4. Click **Continue**. This initializes the project and clones the Git code and builds the Compose application. This:
```